modules/cnv-importing-vmware-vm.adoc

```diff
@@ -1,67 +1,59 @@
 // Module included in the following assemblies:
 //
-// * cnv/cnv_users_guide/cnv-create-vms.adoc
+// * cnv/cnv_users_guide/cnv-importing-vmware-vm.adoc
+[id='Importing_vmware_vm_or_template_{context}']
+= Importing a VMware virtual machine or template
 
-= Importing the VMware virtual machine or template
+You can import a VMware virtual machine or template using the virtual machine wizard.
 
 .Prerequisites
 
-* The VMware virtual machine is powered off.
+* You must create a VDDK image, push it to a private container repository, and add it to the `v2v-vmware` Configmap.
+* You must power off the VMware virtual machine before importing it.
 
 .Procedure
 
-. Click *Workloads* -> *Virtual Machines* from the side menu.
-. Click *Create Virtual Machine* and select *Create with Wizard*.
-. Perform the following procedure in the *Basic Settings* screen:
-.. Click *Provision Source* and select *Import*.
-.. Click *Provider* and select *VMware*.
-.. Click *vCenter Instance* and select *Connect to New Instance* or a saved vCenter instance.
-+
-* If you select *Connect to New Instance*, provide values in the following fields and click *Check and Save*:
+. In the {CNVProductName} web console, click *Workloads* -> *Virtual Machines*.
+. Click *Create* and select *Import with Wizard*.
+. In the *General* screen, perform the following steps:
+.. Select *VMware* from the *Provider* list.
+.. Select *Connect to New Instance* or a saved vCenter instance from the *vCenter instance* list.
 
-** *vCenter Hostname*
-** *vCenter User Name*
-** *vCenter Password*
-+
-The wizard connects to the vCenter instance and saves the credentials.
+.. Select a virtual machine or a template to import.
 +
-If you clear the *Save as New vCenter Instance* check box and click *Check*, the wizard connects to the vCenter instance without saving the credentials.
+The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine or template.
```

Comment:
Operating System is not filled automatically. As a response to the selected VM, Under the Operating system field, such message is displayed: "Select matching for: Red Hat Enterprise Linux 7 (64-bit)" - This is an example for a RHEL-7 VM, that was selected.

Reply:
Yes, I noticed the differences in the video. In 2.1, they were filled in.

So the UI detects the operating system and tells you which one to pick? Why doesn't it just pick it if it knows?

Reply:
Correct.
Probably to allow more freedom to the user.
It is more a design question, that can be addressed to Tomas J./Marek L.

Reply:
I think I will follow this up. I'm curious to know why they decided to do this and what might happen if user chooses incompatible OS.